scripts/utils/cliUtils.ts

```diff
@@ -0,0 +1,21 @@
+import { InvalidArgumentError } from '@commander-js/extra-typings'
+
+export const testVariadicArgs = (
+  currentValue: string,
+  previousValue: string | string[],
```

Comment:
why might we expect the previousValue be an array of strings and not the currentValue?